conformance/framework.go

```diff
@@ -34,28 +34,30 @@ import (
 )
 
 type (
-	DoOperationFunc func() (interface{}, error)
-	CheckResultFunc func(result interface{}) (bool, string, error)
+	doOperationFunc func() (interface{}, error)
+	checkResultFunc func(result interface{}) (bool, string, error)
 )
 
-// AwaitUntil periodically performs the given operation until the given CheckResultFunc returns true, an error, or a
-// timeout is reached.
-func AwaitUntil(opMsg string, doOperation DoOperationFunc, checkResult CheckResultFunc) interface{} {
+// AwaitUntil repeats the given operation until the given checkResultFunc returns true, an error, or a
+// timeout is reached; it then verifies that no error occurred.
+func AwaitUntil(opMsg string, doOperation doOperationFunc, checkResult checkResultFunc) interface{} {
 	result, errMsg, err := AwaitResultOrError(opMsg, doOperation, checkResult)
 	Expect(err).NotTo(HaveOccurred(), errMsg)
 
 	return result
 }
 
-func AwaitResultOrError(opMsg string, doOperation DoOperationFunc, checkResult CheckResultFunc) (interface{}, string, error) {
```

Comment:
I think we can get rid of these two "Await" functions as they're not used and I don't see a need for them.